c/parallel/src/nvrtc/nvjitlink_helper.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#define NVJITLINK_NO_INLINE
+#include <nvJitLink.h>
+#undef NVJITLINK_NO_INLINE
+
+// declare unversioned functions
+
+extern "C" {
+nvJitLinkResult nvJitLinkCreate(nvJitLinkHandle*, uint32_t, const char**);
+nvJitLinkResult nvJitLinkDestroy(nvJitLinkHandle*);
+nvJitLinkResult nvJitLinkAddData(nvJitLinkHandle, nvJitLinkInputType, const void*, size_t, const char*);
+nvJitLinkResult nvJitLinkAddFile(nvJitLinkHandle, nvJitLinkInputType, const char*);
+nvJitLinkResult nvJitLinkComplete(nvJitLinkHandle);
+nvJitLinkResult nvJitLinkGetLinkedCubinSize(nvJitLinkHandle, size_t*);
+nvJitLinkResult nvJitLinkGetLinkedCubin(nvJitLinkHandle, void*);
+nvJitLinkResult nvJitLinkGetLinkedPtxSize(nvJitLinkHandle, size_t*);
+nvJitLinkResult nvJitLinkGetLinkedPtx(nvJitLinkHandle, char*);
+nvJitLinkResult nvJitLinkGetErrorLogSize(nvJitLinkHandle, size_t*);
+nvJitLinkResult nvJitLinkGetErrorLog(nvJitLinkHandle, char*);
+nvJitLinkResult nvJitLinkGetInfoLogSize(nvJitLinkHandle, size_t*);
+nvJitLinkResult nvJitLinkGetInfoLog(nvJitLinkHandle, char*);
+}
```

Comment:
I looked at what's actually being used (below): 10 of 13 APIs.

I'd probably remove the 3 we're not using, with a comment:
```
// nvJitLink APIs used in cccl/c/parallel
```
________
```
      2 src/nvrtc/command_list.h:  nvJitLinkAddData
      1 src/nvrtc/command_list.h:  nvJitLinkComplete
      1 src/nvrtc/command_list.h:  nvJitLinkCreate
      1 src/nvrtc/command_list.h:  nvJitLinkDestroy
      1 src/nvrtc/command_list.h:  nvJitLinkGetErrorLog
      1 src/nvrtc/command_list.h:  nvJitLinkGetErrorLogSize
      1 src/nvrtc/command_list.h:  nvJitLinkGetLinkedCubin
      1 src/nvrtc/command_list.h:  nvJitLinkGetLinkedCubinSize
      1 src/nvrtc/command_list.h:  nvJitLinkGetLinkedPtx
      1 src/nvrtc/command_list.h:  nvJitLinkGetLinkedPtxSize
```